jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/inputs/SelectTests.java

```diff
@@ -1,53 +1,150 @@
 package io.github.epam.material.tests.inputs;
 
-import static io.github.com.StaticSite.inputSelectDisabledPage;
-import static io.github.com.StaticSite.inputSelectMultiplePage;
-import static io.github.com.StaticSite.inputSelectSimplePage;
-import static io.github.com.StaticSite.selectFrame;
+import static io.github.com.MaterialNavigator.openSection;
+import static org.hamcrest.Matchers.containsString;
+import static io.github.com.pages.inputs.SelectPage.*;
+import static org.testng.AssertJUnit.assertTrue;
 
+import com.epam.jdi.light.ui.html.elements.common.Text;
+import com.epam.jdi.tools.Timer;
 import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
 import java.util.Arrays;
 import java.util.List;
 
+/**
+ * To see an example of Select web element please visit https://material-ui.com/components/selects/
+ */
+
 public class SelectTests extends TestsInit {
 
+    private static Timer timer = new Timer(1000L);
+
+    @BeforeMethod
+    public void before() {
+        openSection("Select");
+    }
+
+    @Test
+    public void simpleSelectTest() {
+        timer.wait(() -> simpleSelectExpand.waitFor().visible());
+        simpleSelectExpand.click();
+        selectList.selectItemByText("Henry");
+        simpleSelectField.get(1).is().attr("value", "Henry");
```

Comment:
Can it be used here is().selected like at line 61?

Reply:
here we can't use selected() method because selected applicable only to Dropdown types